Aurelia/Schedule/working-with-appointments.md

```diff
@@ -390,7 +390,7 @@ export class Schedule {
 
 {% endhighlight %}
 
-N> All these CRUD operations on appointments (add/edit/delete) can also be done through the default [context menu](/aurelia/schedule/context-menu#default-menu-options) options **Add Appointment**, **Edit Appointment** and **Delete Appointment** which is available, when context menu settings is enabled within Scheduler.
+N> All these CRUD operations on appointments (add/edit/delete) can also be done through the default [context menu](https://help.syncfusion.com/aurelia/schedule/context-menu#default-menu-options) options **Add Appointment**, **Edit Appointment** and **Delete Appointment** which is available, when context menu settings is enabled within Scheduler.
```

Comment:
Check all this links that you've added https

Reply:
resolved